### ISSUE TITLE
SC2.5.8 Target Size doesn't say what it means

### ISSUE THREAD

**Issue by guyhickling** [w3c/wcag#2319]
This new SC appears to specify a minimum size for pointer targets, from the way the main body of the SC is worded: "The size of the target for pointer inputs is at least 24 by 24 CSS pixels....."

But the Understanding document says "The aim of this Success Criterion is that targets are ideally at least 24 by 24 CSS pixels". The fact that it uses the word "ideally" indicates that it is only a hope, and that the SC does NOT require a minimum size for the target! So what gives?

I'm sure a minimum size was the original intention when this SC was first thought up. But the "Spacing" exemption changes that completely. The result of this exception is that the target size is actually completely irrelevant! The only thing that matters is the sum of the distance across the target plus the distance to the next target. 

I.e. the following all pass: target 24px + space 0px, target 23px + space 1px, and all other combinations all the way down to target 1px + space 23px. In other words the size of the actual target has nothing to do with it. 

In short, whether any given pointer target passes or fails the SC  will ALWAYS depend on the exception, not on what is said in the main body of the SC! 

And now we come to something else: the Understanding actually contradicts the SC, because Figure 2 shows targets that are less than 24 px, with no other targets around, but it still passes them even though they are below 24px in size! So the idea of a minimum 24 pixel size for targets mentioned in the main body of the SC, is a complete red herring that is not true. This will just confuse everybody meeting up with it for the first time.

### Solution
This SC needs to be reworded to say what it means. The wording of the exception should be moved into the main body of the SC instead of what's in the main body now, and the Spacing exception can then be deleted. 

So the SC should read:
> "The distance measured from the farthest point of a target for pointer inputs to the nearest point of a second target is at least 24 CSS pixels......."  [followed by the Inline, Necessary and Legal exceptions]

That will get rid of the fiction that there is a minimum target size, which there isn't at Level AA on 99% of web pages. (There is a minimum at Level AAA, because SC2.5.5 doesn't have a Spacing exception.) It will also make the whole thing much easier to understand and avoid people getting confused. 

And it should be renamed 2.5.8 "Target Spacing" or similar, not Target size.

No doubt people will use many arguments to defend the wording that has been produced for this SC. But the WCAG has a reputation for being abstruse and difficult to understand, and I think this proposed SC, as it stands, will only make that reputation worse!

**Comment by JAWS-test** [comment]
This sounds logical because the spacing exception of 2.5.8 is not a real exception but always applies. 

On the other hand, it may still make sense to formulate the "wrong" exception as an exception. That is, if one sees the WCAG not only as a set of rules that prohibits, but also as a set of recommendations. And it makes a difference whether I say: please use a target size of at least 24x24 px, or whether I say: the size doesn't matter, the main thing is that the distance is right.

**Comment by alastc** [comment]
Hi Guy, we've been through this discussion before based on #1871 and [resolved](https://github.com/w3c/wcag/issues/1871#issuecomment-873135394) to keep the name. There isn't new information here, so I'm going to close this issue.
